x-pack/plugins/siem/public/components/page/network/domains_table/index.tsx

```diff
@@ -0,0 +1,224 @@
+        onChange={this.onChange}
+        sorting={getSortField(domainsSortField, flowTarget)}
+        updateLimitPagination={newLimit =>
+          updateDomainsLimit({ limit: newLimit, networkType: type })
+        }
+        title={
+          <EuiFlexGroup>
+            <EuiFlexItem>
+              <EuiFlexGroup>
+                <EuiFlexItem grow={false}>
+                  <h3>
+                    {i18n.DOMAINS}
+                    <CountBadge color="hollow">{totalCount}</CountBadge>
+                  </h3>
+                </EuiFlexItem>
+              </EuiFlexGroup>
+            </EuiFlexItem>
+            <EuiFlexItem grow={false}>
+              <FlowDirectionSelect
+                id={DomainsTableId}
+                selectedDirection={flowDirection}
+                onChangeDirection={this.onChangeDomainsDirection}
+              />
+            </EuiFlexItem>
+          </EuiFlexGroup>
+        }
+      />
+    );
+  }
+
+  private onChange = (criteria: Criteria) => {
+    if (criteria.sort != null) {
+      const splitField = criteria.sort.field.split('.');
+      const newDomainsSort: DomainsSortField = {
+        field: last(splitField) as DomainsFields,
+        direction: criteria.sort.direction,
+      };
+      if (!isEqual(newDomainsSort, this.props.domainsSortField)) {
+        this.props.updateDomainsSort({
+          domainsSortField: newDomainsSort,
+          networkType: this.props.type,
+        });
+      }
+    }
+  };
+
+  private onChangeDomainsDirection = (_: string, flowDirection: FlowDirection) =>
+    this.props.updateDomainsDirection({ flowDirection, networkType: this.props.type });
+}
+
+const makeMapStateToProps = () => {
+  const getDomainsSelector = networkSelectors.domainsSelector();
+  const mapStateToProps = (state: State) => ({
+    ...getDomainsSelector(state),
+  });
+  return mapStateToProps;
+};
+
+export const DomainsTable = connect(
+  makeMapStateToProps,
+  {
+    updateDomainsLimit: networkActions.updateDomainsLimit,
+    updateDomainsDirection: networkActions.updateDomainsFlowDirection,
+    updateDomainsSort: networkActions.updateDomainsSort,
+  }
+)(DomainsTableComponent);
+
+const getSortField = (sortField: DomainsSortField, flowTarget: FlowTarget): SortingBasicTable => {
+  switch (sortField.field) {
+    case DomainsFields.domainName:
+      return {
+        field: `node.${flowTarget}.${sortField.field}`,
+        direction: sortField.direction,
+      };
+    case DomainsFields.bytes:
+      return {
+        field: `node.network.${sortField.field}`,
+        direction: sortField.direction,
+      };
+    case DomainsFields.packets:
+      return {
+        field: `node.network.${sortField.field}`,
+        direction: sortField.direction,
+      };
+    case DomainsFields.uniqueIpCount:
+      return {
+        field: `node.${flowTarget}.${sortField.field}`,
+        direction: sortField.direction,
+      };
+    default:
+      return {
+        field: `node.network.bytes`,
```

Comment:
nit: you can do: field: 'node.network.bytes' without interpolation

Reply:
Nice catch! Appears to be leftover from when `network` was a variable, thanks! :)